packages/tempus/unit_test/Tempus_UnitTest_IMEX_RK.cpp

```diff
@@ -40,6 +40,8 @@ using Tempus::StepperExplicitRK;
 // Comment out any of the following tests to exclude from build/run.
 #define CONSTRUCTION
 #define STEPPERFACTORY_CONSTRUCTION
+#define STEPPERFACTORY_CONSTRUCTION_GENERAL_WO_PARAMETERLIST
+#define STEPPERFACTORY_CONSTRUCTION_GENERAL_WO_PARAMETERLIST_MODEL
```

Comment:
Why do these defines exist? There is a runtime flag that you can use to selectively execute just a single test from an executable. Maybe this is more convenient?

Reply:
Ah, what does that command look like?  This original came from Irina, and I thought it was useful during debugging.

Reply:
test.exe --test=<my_test_name>

For example, I have a panzer unit test declared with:
```
  TEUCHOS_UNIT_TEST(global_data, builder)
  {
   ...
  }
```

You could run only that one with:
```
./PanzerDiscFE_global_data.exe --test=builder
```